File: app/src/main/java/cn/com/szw/lib/myframework/utils/net/NetEntity.java
package cn.com.szw.lib.myframework.utils.net;


/**
 * Created by Administrator on 2016/7/5.
 */
//@HttpResponse(parser = JsonResponseParser.class)//每一个实体类必须加这句话，别忘了在baseactivity里面 初始化注解 x.view().inject(this);
public class NetEntity<T> {
//    "code":"200",
//            "message":"提示"，
//            "data": "1234"

    private String code;
    private String message;
    private T data;
    private T info;

    public T getInfo() {
        return info;
    }

    public void setInfo(T info) {
        this.info = info;
    }

    public int getCode() {
        return Integer.valueOf(code);
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }
}

File: Flibrary/src/main/java/com/szw/framelibrary/utils/net/NetEntity.kt
package com.szw.framelibrary.utils.net

//@HttpResponse(parser = JsonResponseParser.class)//每一个实体类必须加这句话，别忘了在baseactivity里面 初始化注解 x.view().inject(this);
class NetEntity<T> {
    //    "code":"200",
    //            "message":"提示"，
    //            "data": "1234"

    private var code: String=""
    var message: String=""
    var data: T? = null
    var info: T? = null

    fun getCode(): Int = Integer.valueOf(code)

    fun setCode(code: String) {
        this.code = code
    }
}
